docs/docs/submit-to-starter-library.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Submit to Starter Library
+---
+
+Have you created a Gatsby starter you'd like to add to the [Starter Library](/starters/)? Follow these instructions.
+
+## Steps
+
+There are only two major steps :)
```

Comment:
I'd like to make this a bit more descriptive. Maybe something like this?

```markdown
## How to Add Your Site to the Starter Library

To get your site added to the starter library, follow the two steps below.
```

Saying two "major" steps made me think there might be a lot of minor steps hidden in there. (Also I might be overthinking this.)

Reply:
Might be overthinking, but also no problem with changing it! It's modelled after the site submission and creator submission pages.